src/main/kotlin/com/example/tlitter/requests/user/LoginRequest.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.tlitter.requests.user
+
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
+
+@ApiModel(description = "ログイン情報")
+class LoginRequest(
+    @ApiModelProperty(value = "ユーザーのメールアドレス", example = "a@b.c")
```

Comment:
[nits]
RequestBodyだとswagger上でバリデーションが効くわけではないですが、ドキュメントという意味合いだとnullableかどうかわかりやすくなるのでrequiredの設定はしておいても良さそうに思いました(passwordも同様)
```suggestion
    @ApiModelProperty(value = "ユーザーのメールアドレス", example = "a@b.c", required = true)
```

Reply:
e5e420a にて修正しました